### DISCUSSION TITLE
Welcome to adminerevo Discussions!

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [adminerevo/adminerevo#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mckaygerhard** [comment]
Initial direction of the project explained at https://github.com/adminerevo/adminerevo/issues/17#issuecomment-1606260606 .

**Comment by unix4you2** [comment]
Hi there!
Nice to see this project that continues with Adminer-line project.  You have some hands here if there is something I can help.
Best Regards, John.

**Comment by aoirthoir** [comment]
I am glad to see this project off the ground. Good work!

**Comment by amstre** [comment]
I'm so glad I found out that you continue this project. We use it all the time. Big thanks from developer

**Comment by misternian** [comment]
Thank you very much!

**Reply by LionelLaffineur** [reply]
My pleasure!
I'm looking for help, please read the other topics and see if you would like to contribute.
